app/javascript/components/enrollment/Enrollment.js

```diff
@@ -185,8 +185,12 @@ class Enrollment extends React.Component {
     let delta = this.state.enrollmentState.patient.isolation && 4 === index ? 2 : 1; // number of enrollment steps to skip (1 unless in isolation skipping exposure info on initial enrollment)
     let maxIndex = this.state.enrollmentState.patient.isolation ? 7 : 6; // max number of enrollment steps in the carosel (6 for exposure and 7 for isolation)
 
+    // If we are reviewing, then workflow must have changed
+    const changedWorkflow = this.state.review_mode && lastIndex !== maxIndex;
```

Comment:
nitpick but can we call this `workflowChange`?

Reply:
Change in https://github.com/SaraAlert/SaraAlert/pull/1118/commits/60525138ccead3a44bdd28fc224c25418e209d99.